app/src/main/java/eu/kanade/tachiyomi/ui/manga/MangaScreenModel.kt

```diff
@@ -369,6 +370,45 @@ class MangaScreenModel(
         }
     }
 
+    fun showEditInfoDialog() {
+        val manga = successState?.manga ?: return
+        screenModelScope.launch {
+            updateSuccessState { successState ->
+                successState.copy(
+                    dialog = Dialog.EditInfo(
+                        manga = manga,
+                    ),
+                )
+            }
+        }
+    }
+
+    fun updateMangaInfo(
+        manga: Manga,
+    ) {
+        val state = successState ?: return
+
+        val newTitle = if (manga.title.isBlank()) manga.url else manga.title.trim()
+
+        screenModelScope.launchNonCancellable {
+            updateManga.await(
+                MangaUpdate(
+                    id = manga.id,
+                    title = newTitle,
```

Comment:
The data needs to be saved in a separate column in the database

Reply:
Done with a50fe3ce0914a3bd6318ab5e4eaf58530bb89803